ui/src/static/js/index.js

```diff
@@ -21,17 +28,61 @@ socket.on("refresh", function (data) {
     }, 5)
 
 });
-socket.on("log", function (data) {
+socket.on("ok", function (data) {
+    let  command = data['command'];
+    ClearAll();
+    if (command === "input") {
+        document.getElementById("setInput").classList.add("is-valid");
+    } else if (command === "scalarMultiply") {
+        document.getElementById("scalarMultiplyInput").classList.add("is-valid");
+    } else if (command === "calc") {
+        document.getElementById("calcPolyInput").classList.add("is-valid");
+    }
+});
+function ClearAll(){
+    document.getElementById("setInput").classList.remove("is-invalid");
+    document.getElementById("scalarMultiplyInput").classList.remove("is-invalid");
+    document.getElementById("calcPolyInput").classList.remove("is-invalid");
+    document.getElementById("scalarMultiplyInput").classList.remove("is-invalid");
+    document.getElementById("setInput").classList.remove("is-valid");
+    document.getElementById("scalarMultiplyInput").classList.remove("is-valid");
+    document.getElementById("calcPolyInput").classList.remove("is-valid");
+    document.getElementById("scalarMultiplyInput").classList.remove("is-valid");
+}
+socket.on("error", function (data) {
+   let  command = data['command'];
+   ClearAll();
+    if (command === "input") {
+        document.getElementById("setInput").classList.add("is-invalid");
+    } else if (command === "scalarMultiply") {
+        document.getElementById("scalarMultiplyInput").classList.add("is-invalid");
+    } else if (command === "calc") {
+        document.getElementById("calcPolyInput").classList.add("is-invalid");
+    }
 });
 
 function ClearConsole() {
     socket.emit("clear");
 }
 
-function RunTests() {
-    socket.emit("tests");
-}
 
-function Command() {
-    socket.emit("command");
-}
\ No newline at end of file
+function Command(command) {
+    let json = {
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/PotatoHD404/Laba2/pullRequest?prid=7321542)
```suggestion
        command
```